### PR TITLE
Add v1.6.0 of mattermost-plugin-gitlab to the Marketplace

### PR DESCRIPTION
**Pull request by mickmister**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-plugin-gitlab/issues/337

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -3783,175 +3902,158 @@
     "updated_at": "2019-06-11T19:41:07Z"
   },
   {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
```
Comment (lieut-data) [inline plugins.json:3905]: Are these changes accidental? cc @cpoile, since I think we had hand-tuned this until `master` was updated.

Comment (mickmister) [reply plugins.json:3905]: @lieut-data Certainly not intentional. Should I simply retract the changes related to Calls?

Comment (lieut-data) [reply plugins.json:3905]: Yeah, I think that would be fine. This issue should be short-lived -- thanks, @mickmister!

**Review by lieut-data — APPROVED** [review]

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tested upgrade path
- Tested fresh install path 
-  Sanity checked the plugin start and runs on a Cloud server
- Icon looks good in light and dark theme
- Marketplace record link points to repo as expected
- Version number correctly shows as `1.6.0`

LGTM!

**Review by hanzei — APPROVED** [review]

**Pull request merged**